deepchecks/nlp/text_data.py

```diff
@@ -313,11 +314,24 @@ def calculate_builtin_embeddings(self, model: str = 'miniLM', file_path: str = '
             'open_ai': using the ADA model in the open_ai library. Requires an API key.
         file_path : str, default: 'embeddings.npy'
             The path to save the embeddings to.
+        device : str, default: None
+            The device to use for calculating the embeddings. If None, the default device will be used.
+        long_doc_averaging : str, default 'warn'
+            How to handle long documents (docments that are longer than the model context window).
+            Can be either 'average+warn', 'average', 'truncate' or 'raise'. Averaging is done as described in
+            https://github.com/openai/openai-cookbook/blob/main/examples/Embedding_long_inputs.ipynb
+
+            Currently, applies only to the 'open_ai' model, as the 'miniLM' model can handle long documents.
```

Comment:
Add a todo, i guess for really long documents it cant

Reply:
it can, the problem is just with open_ai